src/module/apps/actor-sheet/npc.mjs

```diff
@@ -28,8 +36,141 @@ export default class DrawSteelNPCSheet extends DrawSteelActorSheet {
       scrollable: [""]
     }
   };
+
+  /** @override */
+  async _preparePartContext(partId, context, options) {
+    await super._preparePartContext(partId, context, options);
+    switch (partId) {
+      case "header":
+        context.monsterKeywords = this._getMonsterKeywords();
+        context.organizationLabel = this._getOrganizationLabel();
+        context.roleLabel = this._getRoleLabel();
+        context.evLabel = this._getEVLabel();
+        break;
+      case "biography":
+        context.motivations = this._getMotivations();
+        break;
+    }
+    return context;
+  }
+
+  /**
+   * Fetches the printable string for the monster's keywords
+   * @returns {string}
+   */
+  _getMonsterKeywords() {
+    const monsterKeywords = ds.CONFIG.monsters.keywords;
+    const formatter = game.i18n.getListFormatter({type: "unit"});
+    const keywords = Array.from(this.actor.system.monster.keywords).map(k => monsterKeywords[k]?.label).filter(k => k);
+    return formatter.format(keywords);
+  }
+
+  /**
+   * Fetches the label for the monster's organization
+   * @returns {{list: FormSelectOption[], current: string}}
+   */
+  _getOrganizationLabel() {
+    const organizations = ds.CONFIG.monsters.organizations;
+    return organizations[this.actor.system.monster.organization]?.label ?? "";
+  }
+
+  /**
+   * Fetches the label for the monster's role
+   * @returns {{list: FormSelectOption[], current: string}}
+   */
+  _getRoleLabel() {
+    const roles = ds.CONFIG.monsters.roles;
+    return roles[this.actor.system.monster.role]?.label ?? "";
+  }
+
+  /**
+   * Fetches the label for the monster's Encounter Value
+   */
+  _getEVLabel() {
+    const data = {value: this.actor.system.monster.ev};
+    if (this.actor.system.monster.organization === "minion") return game.i18n.format("DRAW_STEEL.Actor.NPC.EVLabel.Minion", data);
+    else return game.i18n.format("DRAW_STEEL.Actor.NPC.EVLabel.Other", data);
+  }
+
+  /**
+   * Fetches the options for Motivations
+   * @returns {{list: FormSelectOption[]}}
+   */
+  _getMotivations() {
+    const motivations = ds.CONFIG.negotiation.motivations;
+    const formatter = game.i18n.getListFormatter();
+
+    const currentMotivations = this.actor.system.negotiation.motivations.reduce((arr, motivation) => {
+      motivation = motivations[motivation]?.label;
+      if (motivation) arr.push(motivation);
+      return arr;
+    }, []);
+    const currentPitfalls = this.actor.system.negotiation.pitfalls.reduce((arr, pitfall) => {
+      pitfall = motivations[pitfall]?.label;
+      if (pitfall) arr.push(pitfall);
+      return arr;
+    }, []);
+    return {
+      list: Object.entries(motivations).map(([value, {label}]) => ({value, label})),
+      currentMotivations: formatter.format(currentMotivations),
+      currentPitfalls: formatter.format(currentPitfalls)
+    };
+  }
+
+  /**************
+   *
+   *   ACTIONS
+   *
+   **************/
```

Comment:
The funky syntax here is probably something we should align on. I'm a fan of just:
```js
/* -------------------------------------------------- */
/*   Actions                                          */
/* -------------------------------------------------- */
```
(50 hyphens, three spaces before the label)